pkgs/development/python-modules/selectors2/default.nix

```diff
@@ -20,18 +20,21 @@ buildPythonPackage rec {
   patches = [ ./mapping-import.patch ];
 
   nativeCheckInputs = [
-    nose
+    unittestCheckHook
     psutil
     mock
   ];
 
   checkPhase = ''
+    runHook preCheck
+
     # https://github.com/NixOS/nixpkgs/pull/46186#issuecomment-419450064
     # Trick to disable certain tests that depend on timing which
     # will always fail on hydra
     export TRAVIS=""
```

Comment:
This can probably be set as `TRAVIS = "";` in the derivation to avoid overriding the phase.